src/www/messages/index.js

```diff
@@ -0,0 +1,46 @@
+export * as Amharic from "./am.json";
```

Comment:
This is a bit of a pain to maintain and it's not needed, since you can map the language codes to files.

If you want to map names to code, I think the localization libraries can already do that. But I think it doesn't help a lot for development we want to know exactly what code we are using.

Instead of this file, have an async loader that calls `import()` on demand instead. That's more in line with what we actually use in the code and doesn't require loading all the languages in memory.

Reply:
If you need a list of languages, reuse this instead: https://github.com/Jigsaw-Code/outline-client/blob/b967567f1a62c718df55e1fe9051ea0b5e28ebd1/src/www/ui_components/app-root.js#L433